README.md

```diff
@@ -1 +1,82 @@
-# To_Do_list_interactive-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# TODO_LIST
+
+> It's todo list App that I made it, using HTML, CSS and Javascript.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**## Steps to run the project
+
+- git clone `git@github.com:ZuhRa-HashiMi/To_Do_list_interactive-.git`
+
+- cd `To_Do_list_interactive-`
+
+- npm install
+
+- npm start.**
```

Comment:
- [OPTIONAL] I think it's better to fix this command to correct instructions about how to run your project. Your project will look more professional and create a better UX.

<img width="546" alt="Screen Shot 2022-11-12 at 3 13 52 PM" src="https://user-images.githubusercontent.com/47673311/201488769-48d79801-e962-4454-9fb5-150f717213d0.png">